src/utils/feeChecker.ts

```diff
@@ -1,13 +1,13 @@
 import { BTC } from "../consts";
 import { asset, config } from "../signals";
 
-const relevantMinerFees = (fees) => {
+const relevantMinerFees = (fees: any) => {
     return asset() === BTC
         ? fees.minerFees.quoteAsset
         : fees.minerFees.baseAsset;
 };
 
-export const feeChecker = (pairs) => {
+export const feeChecker = (pairs: any) => {
```

Comment:
Here we should use strong types. The format of the data is known

Reply:
open issue for that